Tests/TestingApps/PaywallsTester/ci_scripts/ci_pre_xcodebuild.sh

```diff
@@ -12,6 +12,7 @@ else
     echo "Replacing API keys on PaywallsTester"
 
     file="$SCRIPT_DIR/../PaywallsTester/Configuration.swift"
+    sed -i.bak 's/private static let apiKey = ""/private static let apiKey = "'$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY'"/g' $file
```

Comment:
Should we still be using the `Simple App` naming for this? I think we decided to change it... Also, I see we have a couple existing API keys: `REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_TESTING` and `REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_DEMOS`. Can we reuse one of those?

Reply:
This isn't a new key, it was defined as well as the others as part of the Xcode cloud build environment. I guess if we're truly done with the others (probably?) we could delete them and the code that uses them, and rename the key while we're at it.

<img width="973" alt="image" src="https://github.com/RevenueCat/purchases-ios/assets/109382862/01e98dcc-0ffb-4c0a-a85b-12b145592db9">